wagtailio/project_styleguide/templates/patterns/components/related-content/related-content.html

```diff
@@ -0,0 +1,13 @@
+{% if related_pages %}
+    <div class="related-content grid">
+        <h3 class="related-content__heading">You might also like</h3>
+
+        <div class="related-content__items">
+            {% for related in related_pages %}
+                <article class="related-content__item">
+                    {% include "patterns/components/streamfields/cards/logo_card_block.html" with value=related classes="logo-card--article" %}
```

Comment:
Yeah I agree - I think we took the name from the miro board and then realised it could be used for other things. I'll add a ticket in the backlog to rename : )